package.json

```diff
@@ -24,8 +27,5 @@
     "name": "JSTransformers Team",
     "url": "http://github.com/orgs/jstransformers/people"
   },
-  "license": "MIT",
-  "dependencies": {
-    "octet": "^3.0.0"
-  }
+  "license": "MIT"
```

Comment:
It would probably make more sense to put dependencies immediately above devDependencies.  It just feels odd to me having "name" and "version" not be the first items.